package.json

```diff
@@ -19,7 +19,7 @@
   "license": "ISC",
   "peerDependencies": {
     "prop-types": "^15.7.2 ",
-    "react": "^15.7.0 || ^16.0"
+    "react": "^18.1.0"
```

Comment:
This change would mean it would no longer work for versions 15 or 16 . if we need to support the last couple versions we will need to have them listed here i believe